src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -83,9 +85,35 @@ std::shared_ptr<ov::Model> add_slices_to_kvcache_inputs(const std::shared_ptr<ov
     return std::make_shared<ov::Model>(model->get_results(), ov::SinkVector{}, new_params);
 }
 
+struct KVAxesPosition {
+    uint32_t batch;
+    uint32_t seq_len;
+};
+
+KVAxesPosition get_kv_axes(const std::string& model_type) {
+    if (model_type == "chatglm")
+        return {1, 0};
+    else if (model_type == "Qwen")
```

Comment:
Could you double check if such type really exists? I believe it may vary but at least should have "Qwen" as prefix

Reply:
Just checked, for some qwen models the model type is "qwen" and for others its "qwen2"